scripts/install-nix-from-closure.sh

```diff
@@ -69,13 +60,17 @@ while [ $# -gt 0 ]; do
         --no-modify-profile)
             NIX_INSTALLER_NO_MODIFY_PROFILE=1;;
         --darwin-use-unencrypted-nix-store-volume)
-            CREATE_DARWIN_VOLUME=1;;
+            (
+                echo "Warning: the flag --darwin-use-unencrypted-nix-store-volume"
+                echo "         is no longer needed and will be removed in the future."
+                echo ""
+            ) >&2;;
```

Comment:
The subshell can be replaced with a braced list instead.

```suggestion
            {
                echo "Warning: the flag --darwin-use-unencrypted-nix-store-volume"
                echo "         is no longer needed and will be removed in the future."
                echo ""
            } >&2;;
```

Though looking at the subsequent lines we're using subshells unnecessarily there too.

Reply:
tentatively added both for testing